packages/main/src/DayPicker.ts

```diff
@@ -664,20 +670,18 @@ class DayPicker extends CalendarPart implements ICalendarPicker {
 
 	/**
 	 * Called by the Calendar component.
-	 * <b>Note:</b> same as for "PageUp"
 	 * @protected
 	 */
 	_showPreviousPage() {
-		this._modifyTimestampBy(-1, "month");
+		this._modifyTimestampByClick(-1, "month");
```

Comment:
The timestamp wasn't properly modified before or the focus wasn't applied?

Reply:
The timestamp when someone is navigating through the calendar ( trying to change the month ) by using the arrow buttons in the header wasnt properly modified.